src/Attribute/ParsedBody.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\RequestModel\Attribute;
+
+use Attribute;
+
+#[Attribute(flags: Attribute::TARGET_PARAMETER)]
+final class ParsedBody implements HandlerParameterAttributeInterface
```

Comment:
Could we name it just `Body`?

Reply:
I think it can be confused with raw `$request->getBody()`, no?

Reply:
I don't think so.